test/js-api/jsapi.js

```diff
@@ -683,48 +710,75 @@ test(() => {
     const instantiateDesc = Object.getOwnPropertyDescriptor(WebAssembly, 'instantiate');
     const instantiate = WebAssembly.instantiate;
     assert_equals(instantiate, instantiateDesc.value);
-    assert_equals(instantiate.length, 2);
+    assert_equals(instantiate.length, 1);
```

Comment:
Per ecma262, `Every built-in Function object, including constructors, has a length property whose value is an integer. Unless otherwise specified, this value is equal to the largest number of named arguments shown in the subclause headings for the function description.`, so it seems 2 was correct for the function can take up to 2 arguments `instantiate(BufferSource bytes [, importObject])`.